lte/gateway/c/core/oai/tasks/amf/amf_app_handler.cpp

```diff
@@ -747,12 +766,14 @@ int amf_app_handle_pdu_session_accept(
   smf_msg->header.pdu_session_id = pdu_session_resp->pdu_session_id;
   smf_msg->header.message_type   = PDU_SESSION_ESTABLISHMENT_ACCEPT;
   // smf_msg->header.procedure_transaction_id = smf_ctx->smf_proc_data.pti.pti;
```

Comment:
you can remove the commented line as it is duplicate with the line you added next.